src/index.html

```diff
@@ -93,7 +93,7 @@
               <i class="fas fa-truck"></i>
             </div>
             <div class="content">
-              <h5>Free shipping</h5>
+              <h5><a href="#">Free shipping</a></h5>
```

Comment:
Just suggestion: nested tags in one line are IMO less readable than broken into separate lines like this:
```
<h5>
  <a href="#">Free shipping</a>
</h5>
```